### PR TITLE
Add description for compareSecret parameters

### LINKED ISSUE #643: CEL doc: compareSecret: parameter description needed
https://github.com/tektoncd/triggers/blob/master/docs/cel_expressions.md

The `compareSecret` functions just say `<string>.compareSecret(string, string, string) -> bool` leaving the reader with no clue as to which parameter is what.... 3 args, but i don't know which one is the secret sub key/prop to be used, which is the secret name and which is the namespace. Its clearly described here: https://github.com/tektoncd/triggers/blob/5348837b14624898ff67b1e7c4b72a6f03982415/pkg/interceptors/cel/triggers.go#L107

### PR DESCRIPTION
**Pull request by dibyom**
# Changes

Fixes #643

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: docs/cel_expressions.md [hunk 1]
```diff
@@ -283,6 +283,7 @@ interceptor.
     <td>
       Constant-time comparison of strings against secrets, this will fetch the secret using the combination of namespace/name and compare the token key to the string using a cryptographic constant-time comparison..<p>
       The event-listener service account must have access to the secret.
+      The parameters to the function are 1. the key within the secret, 2. the secret name, and 3. the namespace for the secret.
```
Comment (bigkevmcd) [inline docs/cel_expressions.md:286]: Could maybe mention that the namespace is optional, and defaults to the same namespace as the event-listener?

Comment (dibyom) [reply docs/cel_expressions.md]: done.

**Review by savitaashture — APPROVED** [review]
/lgtm

**Pull request merged**